stock_request/tests/test_stock_request.py

```diff
@@ -451,7 +492,7 @@ def test_stock_request_order_validations_06(self):
         expected_date = fields.Datetime.now()
         child_expected_date = "2015-01-01"
         vals = {
-            "company_id": self.company_2.id,
+            "company_id": self.main_company.id,
```

Comment:
Why this change ?

Reply:
I probably think this `test_stock_request_order_validations_06` was the same test as [test_stock_request_order_validations_05](https://github.com/OCA/stock-logistics-warehouse/pull/1637/files#diff-aea988ceb44324ce797bd2eaf94bc771996f7db0ef6308404d2bf431570ec55cR430), but maybe not... I'll probably restore this test and create a new one changing the doc-string seems to be outdated.

Thanks